lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -16,8 +16,7 @@ mapped_vni:
     kind: int
     config_get: 'show running vlan'
     config_get_token: ['/^vlan <vlan>$/', '/^vn-segment (\d+)$/']
-    # In the cli 'vn-segment' config is not applied untill the 'end'
-    # statement is issued.
+    # The 'vn-segment' config is not applied until the 'end' statement is processed
```

Comment:
No, and yes. :-)

From what I see, after removing the 'end'(s), all of the positive tests seem to work (vlans get created with the correct properties); however, the negative tests fail because they don't see a RuntimeError. I add the 'end' back and they get the RuntimeError again.